### PR TITLE
feat: add typical workday English block

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by moT01** [comment]
Only glanced at the diff here - we've been using the ID's for the filenames, for the same reason as we do with RWD. I would do that, and probably rename the exam challenge to just another task - otherwise it could get confused with the final exam.

**Comment by naomi-lgbt** [comment]
>  we've been using the ID's for the filenames, for the same reason as we do with RWD.

I noted this in my earlier PR, but I'll run a script to rename them all once we're happy with the changes (because it is a THOUSAND times easier to apply reviews when the file names are still readable haha)

**Comment by moT01** [comment]
> I'll run a script to rename them all once we're happy with the changes

Sounds good 👍 You're going to push that to this PR then or make a new one for that?

**Comment by naomi-lgbt** [comment]
Yeah I'll push to this one after it's approved. :3

**Review by moT01 — COMMENTED** [review]
This is looking good @naomi-lgbt 🎉

I just made notes of what I found...

I noticed a number of blanks that had two words in them. For instance, `have to` would be used as a single blank. I kind of like it actually, so I’m not sure if we should change them - but typically, we’ve been using one word per blank. I think we should probably keep it that way for consistency. Tasks 14, 27, 53, 65, and 115 have this issue.

There’s a couple tables in the markdown where it's hard to distinguish the rows and columns. I think we might be better off with a list - not sure. Task 2 and 39 have this issue.

Task 4:
Should the times not be in backticks?

Task 5:
The last blank doesn’t seem to make sense. I think it’s supposed to be `reply`.

Task 7:
Needs a space in the FITB sentence - or just delete the second half.

Task 14:
Don’t use `ex:` for `example`

Task 47:
Should the first `security` be capitalized? I capitalized it on one of the challenges in 1.1.

Task 73:
Doesn’t have a question.

Task 91 and 93:
The fill in the blank sentence doesn’t seem right. Seems like it should be `Compliance is when we put specific rules, policies, and regulations in place that are important for our to practice.
`
^ Added `in place` and changed the first `to` to `for` - maybe needs another comma in there somewhere, too.

Task 92:
The first answer passes, but it’s wrong and the same as the second answer. It needs to be something else - maybe `To ensure fairness and honesty`

Task 116 & 117:
Some ugly formatting on the FITB sentences. I kinda like the style of the challenge - just some rough formatting.

Task 122:
The answer doesn’t seem correct, `She only works on Mondays` - it is probably supposed to be option 2.

Tasks 127:
`bugs` should probably be in backticks

I don't think any of these issues are too critical, as we have another chance to catch them when we add the animations - fix the ones you want @naomi-lgbt.

**Comment by naomi-lgbt** [comment]
> For instance, have to would be used as a single blank. I kind of like it actually, so I’m not sure if we should change them

IMHO, it makes sense to use a single blank when we're explicitly teaching an expression. But I can fix this to be individual blanks if we'd prefer.

> Task 4:
> Should the times not be in backticks?

I put them in backticks because they tie directly to the audio. If we think otherwise, happy to change it.

> Task 47:
> Should the first security be capitalized? I capitalized it on one of the challenges in 1.1.

I'd argue no. It's not something I typically see capitalised in normal conversation. "I work in security" vs "I work in Security".

> Task 116 & 117:
> Some ugly formatting on the FITB sentences. I kinda like the style of the challenge - just some rough formatting.

This should look better with the improved formatting that Oliver implemented.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/task-94.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Note on `Harm` and `Harmful`:
+
+`Harm` means damage or injury to someone or something. It refers to causing a negative effect or putting someone or something at risk. 
+
+For example, `Touching hot surfaces can cause harm,` means that touching hot surfaces can lead to injury or damage.
+
+`Harmful` is an adjective used to describe something that can cause harm. It means dangerous or risky in a way that could lead to damage or injury. 
+For example, `Smoking is harmful to health,` indicates that smoking can cause health problems or damage.
+
+These terms are often used to discuss safety and risks, especially in contexts like health, security, and the environment.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/task-94.md:24]:
```suggestion
A `security incident` is an event that may be harmful to the safety or security of a company or organization. For example, `A security incident like a stolen password needs to be reported immediately.`

Note on `Harm` and `Harmful`:

`Harm` means damage or injury to someone or something. It refers to causing a negative effect or putting someone or something at risk. For example:

 `Touching hot surfaces can cause harm,` means that touching hot surfaces can lead to injury or damage.

`Harmful` is an adjective used to describe something that can cause harm. It means dangerous or risky in a way that could lead to damage or injury. For example:

 `Smoking is harmful to health,` indicates that smoking can cause health problems or damage.

These terms are often used to discuss safety and risks, especially in contexts like health, security, and the environment.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/task-94.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+# --question--
+
+## --text--
+
+What is a 'security incident'?
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/task-94.md:30]:
```suggestion
What is a `security incident`?
```

**Review by nieldakarla — CHANGES_REQUESTED** [review]
This looks great Naomi! =D
I've made a few minor changes, correcting some spaces, adding a few backticks.

**Review by nieldakarla — APPROVED** [review]
LGTM

**Review by moT01 — APPROVED** [review]

**Pull request merged**